TombEngine/Scripting/Internal/TEN/Strings/DisplayString/DisplayString.cpp

```diff
@@ -181,6 +186,16 @@ std::string DisplayString::GetKey() const
 	return s.m_key;
 }
 
+void DisplayString::SetFlags(sol::table const& flags) {
```

Comment:
Please follow TEN conventions. ;) Braces should always be on separate lines, `const` should be placed before `sol::table&` (all of scripting had this wrong from the beginning).